.github/workflows/deploy-frontends-to-staging.yml

```diff
@@ -229,6 +229,12 @@ jobs:
           echo "service: staging" >> app.yaml
           echo "includes:" >> app.yaml
           echo "  - .env.yaml" >> app.yaml
+          echo "automatic_scaling:" >> app.yaml
+          echo "  cool_down_period: 120s" >> app.yaml
+          echo "  min_num_instances: 1" >> app.yaml
+          echo "  max_num_instances: 5" >> app.yaml
```

Comment:
Hi @BenjaminSsempala , thanks for this.

While the space itself doesn't cause issues, it's generally considered good practice to remove unnecessary spaces in configuration files or YAML syntax to keep the content clean and consistent. Using this specific part as an example for other areas, we could modify the command to: `echo "max_num_instances: 5" >> app.yaml`

Otherwise, this would not change the functionality of the command in terms of appending the string to the file.

Thanks again.

Reply:
Hey @Baalmart 
This follows the app.yaml syntax, the space is intended

Reply:
thanks @BenjaminSsempala